vscode-dotnet-runtime-extension/src/extension.ts

```diff
@@ -338,6 +344,70 @@ export function activate(vsCodeContext: vscode.ExtensionContext, extensionContex
         return pathResult;
     });
 
+    const dotnetUninstallPublicRegistration = vscode.commands.registerCommand(`${commandPrefix}.${commandKeys.uninstallPublic}`, async () =>
+    {
+        const existingInstalls = await InstallTrackerSingleton.getInstance(globalEventStream, vsCodeContext.globalState).getExistingInstalls(true);
+
+        const menuItems = existingInstalls?.sort(
+        function(x : InstallRecord, y : InstallRecord) : number
+        {
+            if(x.dotnetInstall.installMode === y.dotnetInstall.installMode)
+            {
+                return x.dotnetInstall.version.localeCompare(y.dotnetInstall.version);
+            }
+            return x.dotnetInstall.installMode.localeCompare(y.dotnetInstall.installMode);
+        })?.map(install =>
+        {
+            return {
+                label : `.NET ${(install.dotnetInstall.installMode).toUpperCase()} ${install.dotnetInstall.version}`,
+                description : `${install.dotnetInstall.architecture ?? ''} | ${install.dotnetInstall.isGlobal ? 'machine-wide' : 'vscode-local' }`,
+                detail : `Used by ${install.installingExtensions.join(', ')}`,
+                iconPath : install.dotnetInstall.isGlobal ? new vscode.ThemeIcon('shield') : new vscode.ThemeIcon('trash'),
+                internalId : install.dotnetInstall.installId
+            }
+        });
+        const chosenVersion = await vscode.window.showQuickPick(menuItems, { placeHolder: 'Select a version to uninstall.' });

```

Comment:
If there are no versions installed, show a notification when command is invoked rather than empty list.

Reply:
I think this is a valid behavior but I'm not sure I prefer this behavior. Is this how other uis in devkit work? I'm curious to hear what other's opinions are.

Reply:
And empty quick pick is not intuitive to user that if the generating list for quick pick failed to work correctly by extension or there are no items to display. You can choose to show it whichever way you want but you should notify user that there are no installations.

Reply:
That's a good point. I'll follow this suggested change. Thanks for looking at this PR!